backend/config/urls.py

```diff
@@ -30,4 +30,5 @@
     path('example/', views.example),
     path('example/<example_id>', views.example),
     path('map', views.map_page),
+    path('timeline', views.timeline_page, name='timeline'),
```

Comment:
should be `path('timeline/', views.timeline_page, name='timeline'),`
Missing the slash, which gets implicitly added. The url above, map, is missing the same, but that has been fixed in `origin/main` 🎵